ar71xx: TL-WR1043N v4/v5: add orange WAN LED support, fix inconsistency in constant name, fix port mapping

majik-sheff opened the pull request:
Add missing definitions for the orange WAN LED on the
TL-WR1043N(D) v4 and v5.

Change the name of a MAC address offset constant to
make it consistent with the format of the
existing constants.

Signed-off-by: Tim Thorpe <tim@tfthorpe.net>

majik-sheff committed:
ar71xx: TL-WR1043N v4/v5: add orange WAN LED support

Add missing definitions for the orange WAN LED on the
TL-WR1043N(D) v4 and v5.

Change the name of a MAC address offset constant to
make it consistent with the format of the
existing constants.

Signed-off-by: Tim Thorpe <tim@tfthorpe.net>

blogic commented:
the patch renames the mac addr offset define, yet does not say that it does so in the commit description.
please either remove that change or add it to the description.

majik-sheff commented:
I made the suggested edit to the commit message.

majik-sheff commented:
Any chance of getting this one re-reviewed?  I know you guys are busy but this one seems pretty small.  It'd be nice to scratch it off.

Commit:
ar71xx: TL-WR1043N v5: fix mapping of LAN ports to labels on housing.

Reversed the numbering of the LAN ports in Luci so as to match the port numbers on the housing.

Signed-of-by: Tim Thorpe<tim@tfthorpe.net>
